packages/devtools-source-map/src/utils/fetchSourceMap.js

```diff
@@ -28,9 +30,12 @@ function _setSourceMapRoot(
   absSourceMapURL: string,
   source: Source
 ) {
-  // No need to do this fiddling if we won't be fetching any sources over the
-  // wire.
-  if (sourceMap.hasContentsOfAllSources()) {
+  // No need to do this fiddling if we won't be fetching any sources
+  // over the wire.  However, we do still want to if any of the source
+  // URLs are relative.  What's difficult is that we want to pretend
+  // that some non-URLs, like "webpack:/whatever", are actually URLs.
+  if (sourceMap.hasContentsOfAllSources() &&
+      sourceMap.sources.every(sourceName => URL_ISH.test(sourceName))) {
```

Comment:
this is minor, but can we move this to a temporary variable so that it is a bit more self documenting

```js
const allURLsAreAbsolute = sourceMap.sources.every(sourceName => URL_ISH.test(sourceName)
```